NestJS/src/Event/event.service.ts

```diff
@@ -14,15 +14,12 @@ export class EventService {
   async createEvent(event: Event): Promise<Event>{
     return new Promise(function(resolve, reject){
       google.run(event, function(err, response) {
-        if(err){
-          resolve(err);
-        }
-        else{
-          const eventOfDatabase = event.body;
+        let eventOfDatabase = event.body;
+        if(!err){
           eventOfDatabase.id_event = response.id;
           getRepository(Event).insert(eventOfDatabase);
-          resolve(eventOfDatabase);
         }
+        resolve((err) ? err : eventOfDatabase);
```

Comment:
i think it is more clear if on error we will call `reject` instead of `resolve` so in case of error promise will be rejected

Reply:
What about function of removeEvent?? if I will use whether reject or resolve in the if-else, then it repeats.

Reply:
I mean in the callback function google.run.

Reply:
Moreover, if I will use reject, then it will not send pythonists about error, but it will show error in the cmd.

Reply:
```
async createEvent(event: Event): Promise<Event>{
    return new Promise(function(resolve, reject){
      google.run(event, function(err, response) {
        let eventOfDatabase = event.body;
        if(!err){
          eventOfDatabase.id_event = response.id;
          getRepository(Event).insert(eventOfDatabase);
        }
        err ? reject(err) : resolve (eventOfDatabase);
      });
    })
  }
```
and wrap in try-catch call of createEvent